<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>

  Adithi Rao


  | When Whales Walked - A Review of Phylogenetics, Development and Evolution of Marine Mammals

</title>
<meta name="description" content="This is a placeholder description.
">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

<!-- Styles -->

<link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22></text></svg>">

<link rel="stylesheet" href="/assets/css/main.css">
<link rel="canonical" href="/blog/2021/Marine-Mammal/">

<!-- JQuery -->
<!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>


<!-- Theming-->






    
<!-- MathJax -->
<script type="text/javascript">
  window.MathJax = {
    tex: {
      tags: 'ams'
    }
  };
</script>
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.1.2/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
    <div class="container">
      
      <a class="navbar-brand title font-weight-lighter" href="/">
       Adithi Rao
      </a>
      
      <!-- Navbar Toggle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/">
              about
              
            </a>
          </li>
          
          <!-- Blog -->
          <li class="nav-item active">
            <a class="nav-link" href="/blog/">
              blog
              
            </a>
          </li>
          
          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          
          
          
          
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      

<div class="post">

  <header class="post-header">
    <h1 class="post-title">When Whales Walked - A Review of Phylogenetics, Development and Evolution of Marine Mammals</h1>
    <p class="post-meta">July 19, 2021</p>
  </header>

  <div align="justify">
    <article class="post-content">
      <h3 id="introduction">Introduction</h3>

<div class="float-right">
<img class="img-fluid rounded" src="/assets/img/Marine_F1.png" width="350" />
</div>

<p>Over 100 living mammalian species depend on the ocean for most, if not all of their needs. A recurrent theme in the evolution of tetrapods is the return to an aquatic habitat. While amphibians never completely parted with the water, most other tetrapod groups have various representatives that are seen to have returned to the water, to some degree. Extant marine mammals include a diverse cohort of species, that have representatives in three broad orders. The order Carnivora contains the Pinnipeds (sea lions, walruses and seals), sea otters and polar bears. The order Cetacea includes animals such as whales, porpoises and dolphins, and finally the order Sirenia consists of sea cows like manatees and dugongs. Extinct marine mammals were no less diverse, including hippopotamus-resembling desmostylians, the strange bear-like carnivore Kolponomos, and even aquatic sloths like Thalassocnus <a href="https://www.researchgate.net/publication/281874441_Marine_Mammals_Evolutionary_Biology_Third_Edition">[1]</a>.</p>

<p>Despite being so taxonomically diverse, all three groups of mammals are influenced and unified in some way by the ocean <a href="https://www.wiley.com/en-us/Marine+Mammal+Biology%3A+An+Evolutionary+Approach-p-9781444311334">[2]</a>. Because the unification is so large and so drastic, it is extremely useful to study the similarities and differences among these various marine mammal groups, that have so seamlessly adapted to life in the water. Their skin changed for an increased amount of insulation (through acquiring blubber/fur), as the temperature dipped. In addition, their ears, nose and limbs have also changed, to give them the ability to not only swim, but also comfortably exist underwater. Among sensory adaptations, the most discussed is the high frequency sound used by whales, for echolocation. The other groups of marine mammals have been able to develop an acute sense of smell, sensitive nerve fiber whiskers that they utilize as tactile organs, and even heavily lobulated kidneys, to cope with the high salt concentration in the water <a href="https://www.researchgate.net/publication/281874441_Marine_Mammals_Evolutionary_Biology_Third_Edition">[1]</a>! These are just a few of the many, many fascinating changes that these organisms underwent while transitioning from land to water.</p>

<p>While sirenians and pinnipeds have also undergone morphological, developmental and functional changes, we can see that the mechanisms through which these changes occurred, are significantly different. Besides, an important challenge in marine mammal studies, as well as in the process of science in general, is the lack of ancient data. Unearthing new cetacean data has opened a multitude of doors in marine mammal studies, but one must be careful not to make too many generalizations in extrapolating cetacean data to the other two marine mammal groups <a href="https://doi.org/10.1038/ng.3198">[3]</a>. Cetaceans, pinnipeds and sirenians have undergone independent adaptation events and evolved very similar physiological features and body plans. The molecular and genetic mechanisms underlying many of these key adaptations have been receiving lots of attention, and yet are vastly unknown <a href="https://doi.org/10.1186/s12862-016-0682-4">[4]</a>. Nonetheless, there are several studies and researchers who have acknowledged that the first step to discovering these mechanisms, is finding the precise relationship between these animals. For this reason, I felt it was pivotal to attempt to reconstruct a phylogenetic tree of my groups of interest, and perform an inference and analysis.</p>

<h3 id="methods">Methods</h3>

<p>In general, systematics is a great area to explore since one can not only use it to understand evolutionary contexts, but also how organisms change over time, the direction of the changes and the frequency with which the changes transpire. It also makes it much easier to compare the descendants of one common ancestor and look for patterns of origin, which is what I am interested in. Though phylogenetic studies in the literature are quite common <a href="https://doi.org/10.1038/ng.3198">[3]</a> <a href="https://doi.org/10.1073/pnas.96.18.10261">[5]</a> <a href="https://doi.org/10.1186/1471-2148-8-216">[6]</a>, studies that combine all three marine mammal groups and their terrestrial relatives are few. To map out this relationship, I performed a phylogenetic reconstruction of the extant marine mammal lineages, along with their land- living counterparts.</p>

<p>To build the tree, I chose 27 species of marine mammals; I made sure to have a representative set of species, evenly distributed over all three groups. For cetaceans, I used 7 species, for sirenians, 4, and for pinnipeds, I used 6 – I made sure to include individuals from a variety of families. I also chose their corresponding related terrestrial animals (Table 1), based on existing literature <a href="https://www.wiley.com/en-us/Marine+Mammal+Biology%3A+An+Evolutionary+Approach-p-9781444311334">[2]</a>. I obtained the gene sequences for all 27 marine species as well as my out-group, from GenBank, and downloaded it in FASTA format. The phylogenetic analysis was performed based on between 400 and 1500 base pairs (bp) of mitochondrial DNA, cytochrome b gene sequence data, for all 28 individuals in the tree (including the out-group).</p>

<div class="container">
<div class="row">
<div class="col">
<figure class="text-center">
  <img class="center-block" src="/assets/img/Marine_T1.png" width="550" />
</figure>
</div>
</div>
</div>

<p>Mitochondrial DNA is extremely widely used as a marker in ancient DNA studies, especially for marine mammals. Since every cell contains thousands of copies of the mitochondrial genome (as opposed to one copy of the nuclear genome), amplification potential through polymeric chain reaction is greatly increased. Mitochondrial DNA also evolves under neutrality, and is often used to make demographic inferences. In neutrally evolving markers, the diversity is expected to be directly proportional to the effective population size, so changes in diversity are usually inferred to reflect demographic changes <a href="https://doi.org/10.1038/ng.3198">[3]</a>).</p>

<p>In addition to the marine species, one out-group was chosen – <em>Ornithorhynchus anatinus</em> (Platypus) – to note the relative positions of the other 27 species. Numerous marine mammal studies in the past <a href="https://doi.org/10.1038/ng.3198">[7]</a> have used the platypus and similar species as an out-group. Further, I felt that this species was quite distant from the in-group, while still being adequately related – this took some trial and error. The out-group also needed to be rooted manually, which might be of interest to know. The out-group not rooting automatically is quite normal, and often has to do with a lack of data.</p>

<p>To build a phylogenetic tree, I used the software MEGAX (Molecular Evolutionary Genetics Analysis). MEGAX offers a variety of different options and customizations, so that one can choose the methods that are most fitting for their respective project. After downloading the FASTA sequences, I needed to align them. For the alignment step, I decided to go with the DNA Muscle alignment option. This made sense to do, since this method can generate multiple alignments of amino acids and nucleotide sequences. It also has a very high speed and accuracy level, in comparison to MAFFT, CLUSTALW, and other available alignment options.</p>

<p>In terms of the type of tree I wanted to build, the choice was Maximum Likelihood Estimation. This method helps by providing the probabilities of the sequences, given their evolution, on a particular constructed tree. By this logic, the more probable the sequences, the more the tree is preferred. It aims to find the tree that gives you the highest probability of the observed characteristics; it is a very robust and commonly used method, even in the existing marine mammal literature. This choice also allows one to infer the branch length (the time axis), which we would be unable to do with other methods like Maximum Parsimony, which would just provide the topology without any temporal information. Though I did find the branch lengths in my analysis, I have left them out of this report since the focus here is on the relationships between species, as opposed to the timeline of divergence.</p>

<p>The model I used was the General Time Reversible Model; the genome itself is not a monolith, and I felt that this method was best suited, with my limited understanding of it, as well as past literature. Not all positions of the genome evolve at the same rate, so for the rates among sites, Gamma Distributed with Invariant Sites (G + I) was chosen. The logic behind choosing this model was that not all sites would mutate, but the ones that do, correspond to a gamma distribution. The heuristic method chosen was the default setting – nearest neighbour interchange (NNI) – as were all other options.</p>

<p>After making these choices, I bootstrapped my tree with a value of 500. This is a very important step to ensure that the resulting tree is computationally robust, especially when constructing trees with not many species and not enough sequences and/or base pairs. This bootstrapping process produces a value at each node, which indicates what percentage of times the same branch was observed in the same location, each time the tree was reconstructed in every subsequent iteration (i.e., the percentage of times the same nodal relationship was recovered). Essentially, it assesses the confidence associated with each node or speciation event in the observed tree, and depicts the one final consensus tree that is most likely.</p>

<h3 id="inferences-and-discussion">Inferences and Discussion</h3>

<p>Phylogenetic trees like this one (Figure 2), help us gauge the evolutionary relationship among species or other entities that are assumed to have a common ancestor. In these trees, each node, which is a taxonomic unit, represents the most recent common ancestor of its descendants. A relatively high amount of sequences analysed (400 – 1500 bp), the use of the maximum likelihood method, and even the introduction of the gamma parameter in the analysis, resulted in the phylogenetic tree shown in Figure 2. Because the method used here is Maximum Likelihood, the simplifying assumption made is that the species evolve independently, once initially diverged.</p>

<p>One systematic way of analysing this tree, would be to examine each group individually. Cetaceans (whales, dolphins and porpoises), are at the very bottom of the tree (Box A, Figure 2). We can see that their relative relationships are quite strong, since all the whales, porpoises and dolphins are part of the same clade. However, though their closest terrestrial relatives are thought to be artiodactyls like cattle and deer, the relationship is not reflected in the tree. Instead, these terrestrial mammals are a part of the above clade, with the pinnipeds and sirenians (Box B, Figure 2).</p>

<div class="float-right">
<img class="img-fluid rounded" src="/assets/img/Marine_F2.png" width="450" />
</div>

<p>The sirenians (manatees and dugongs) are also adequately represented in the tree, since they are all not only adjacent to one another in the same clade, but also have very high bootstrap numbers. Additionally, their closest living relatives are rock hyraxes and elephants. While the tree shows an impressively high confidence for their association with hyraxes (Box C, Figure 2), the elephant seems to be placed quite far away.</p>

<p>Finally, the pinnipeds (sea lions, seals, walruses) are at the very top of the tree. Other than the walrus, they are all shown to be part of the same clade, and rightly so, as more concrete studies have shown. They are not only in the same clade as each other, but also as their terrestrial relatives – skunks and otters (Box D, Figure 2). However, brown bears, walruses and polecats, though together, are in the cetacean clade, below. Unsurprisingly, the bootstrap value of this clade is a mere 8, so it is definitely not a very likely relationship.</p>

<p>On examining the tree, it is clear that there are quite a few discrepancies. I felt it would be more useful to use the tree I got with the available data and explain the visible discrepancies, as opposed to attempt to create the “perfect” tree. For one, it seems that a lot of nodes have bootstrap values much lower than 100. This could be due to multiple plausible reasons. Firstly, insufficient data – between 400 and 1500 bp of nucleotides were used, which might not be sufficient for a group that is this genetically similar. Another reason for some clades having low bootstrap values could potentially be because more than one clade was equally distant from other groups, so the software was not able to assign numbers effectively. It is also possible that a low bootstrap value means that those species do not actually belong together in the same clade, which is definitely true in this case (since similar trees have already been built and are known to be true <a href="https://doi.org/10.1073/pnas.0602920103">[8]</a>). We can also see that the out-group does not have an associated bootstrap value at all; this is because it was manually rooted by force.</p>

<p>A critical concept while constructing phylogenetic trees, especially in a study of this nature, is homology. Homology is essentially a similarity in traits, owing to a shared ancestor. Two features are characteristically homologous, if the last common ancestor possessed it. For instance, the flippers possessed by seals and sea lions are homologous, since we can see that they shared a very recent common ancestor, which also possessed flippers (Point 1, Figure 2). Homoplasy, on the other hand, is the opposite; the flippers possessed by a seal and by a whale are homoplasious, since their last common ancestor (which is far behind, since they are quite distantly related) lacked them (Point 2, Figure 2). Homoplasy comes about in two distinct ways; one, independent evolution of the same features (convergence) or two, reversal, loss of the derived feature, with the re-establishment of the ancestral feature, a second time. Through fossil evidence and gene sequencing, we know that in the case of cetaceans and pinnipeds, the former theory holds true – their flippers evolved completely independently, and their similarities are homoplasious by convergent evolution  <a href="https://www.researchgate.net/publication/281874441_Marine_Mammals_Evolutionary_Biology_Third_Edition">[1]</a><a href="https://doi.org/10.1002/ar.20545">[10]</a>. Over the past few decades, it has become quite well established that marine mammals are crucial biological models, especially in the study of convergent evolution and aquatic adaptations. Despite coming from very different ancestors, all three groups of mammals have adapted to life in similar habitats – they independently recolonized the aquatic environment <a href="https://doi.org/10.1186/s12862-016-0682-4">[4]</a>.</p>

<p>Although there are a multitude of genomic routes to reach the same phenotype and morphological outcome, it has been suggested that convergent evolution often causes genomic changes to be reproducible, meaning that convergent phenotypic traits may commonly occur due to the same genetic modifications. In fact, convergence in phenotypes has been linked to identical replacements of single amino acids within the encoded product of a gene, independently in unrelated taxa. Studies in the past have explored this, and found that convergent amino acid substitutions are indeed widespread throughout the genome, and a subset of these substitutions are in genes associated with the marine phenotype, evolving under positive selection. Parallel changes seen in coding genes mapping to the same amino acid site in more than one marine mammal lineage were also found to be widespread in the genome <a href="https://doi.org/10.1038/ng.3198">[7]</a>. The only way to properly discern and compare these findings in each lineage, is through a holistic approach – a good way to understand this, is through an example.</p>

<p>The largest focus in marine mammal evolution has always been skeletal adaptations, as well as hind-limb loss. I addressed cetacean hind-limb loss in my presentation for Milestone 1 (allowing us to come full circle!); however, that was from a single perspective. In reality, it can be studied holistically, from multiple perspectives – paleontological, developmental and even functional. From a paleontological perspective, hind-limb reduction and loss has been greatly documented, and each specific morphological stage can be clearly linked to different locomotor modes. While early cetaceans like Pakicetus had large feet that they used for movement in the water, later cetaceans developed long and muscular tails for propulsion; over time, their hind- limbs because rudimentary. From a functional perspective, studies show that the loss of hind- limbs aided in streamlining the body; otters, for example, are great examples of functional models for the different evolutionary stages of marine mammal evolution. Essentially, hind- limb reduction led to more efficient swimming, and selection for loss of hind-limbs must have been strong, especially in the presence of predators <a href="https://doi.org/10.1073/pnas.0602920103">[8]</a>.</p>

<div class="float-right">
<img class="img-fluid rounded" src="/assets/img/Marine_F3.png" width="300" />
</div>

<p>From a developmental perspective too, hind- limb loss has been studied extensively; yet, many mechanisms still remain unknown. Nonetheless, there are some parallels that can be drawn to studies performed on other tetrapods. We know that in amniote embryos, there are two signaling centers that control limb growth – the apical ectodermal ridge (AER) and the zone of polarizing activity (ZPA). In amniotes, Fgf4, 8, 9 and 17 mediate signaling activity of the AER. The ZPA, on the other hand, is characterized by the expression of Sonic hedgehog (Shh). Shh in turn, is partly regulated by a transcription factor called Hand2 <a href="https://doi.org/10.1073/pnas.0602920103">[8]</a>.</p>

<p>In mice, chicks and other tetrapods, Hand2 is initially expressed throughout the limb, and is later limited to the posterior edge of the limb buds, where Shh is continually needed. For marine mammals that have lost their hind- limbs, though, Thewissen et al. found that there is a deviation from this regular mechanism. It was found that these animals display a hind- limb bud with a distinct AER around Carnegie 13 stage (standardized 23 stage development chronology), which is lost by Carnegie 15 (Figure 3).</p>

<p>After AER loss, consequently the hind-limb bud also reduces in size, and nearly disappears. Since the AER is initially present, the degeneration of it is probably not the primary reason behind the absence of the hind-limbs. Instead, it may just reflect an underlying deficiency in cell signaling, perhaps in the ZPA. In fact, it was found that the real source of the change was the upstream regulator of Shh, Hand2. Hand2 is present in the forelimb region, but undetectable in the area of the developing hind-limb; this may be the underlying reason for the failure of ZPA establishment and hind-limb formation <a href="https://doi.org/10.1073/pnas.0602920103">[8]</a>!</p>

<h3 id="conclusion">Conclusion</h3>

<p>Though one write-up is not enough to do justice to a topic like this one, it is extremely fascinating to try and understand the changes these animals underwent at various levels, to become what they are today. The water did act as a new niche for these animals, but in doing so, it had so many other implications, leading to the emergence of completely different species – morphologically and otherwise. Despite the advances in understanding marine mammals (especially cetaceans), it remains unclear which processes caused this pattern of land-to-water transition: Why did so many lineages enter the oceans? Was it to discover new food sources, for refuge from danger, or even to deal with their own weight <a href="https://doi.org/10.1007/s12052-009-0135-2">[9]</a>?</p>

<p>Even though a multitude of questions remain unanswered, the rich fossil record, new molecular tools and research workflows, can now aid and enrich several subfields of molecular and evolutionary science, including developmental biology, comparative anatomy and molecular systematics <a href="https://doi.org/10.1007/s12052-009-0135-2">[9]</a>. As these processes grow deeper, the hope is that they will eventually allow one to answer the above numerous questions, and finally determine the phenomena that drove marine mammals back into the water – a niche they spent millennia trying to escape in the first place!</p>


    </article>
  </div>

  

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    &copy; Copyright 2021 Adithi  Rao.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>.

    
    
  </div>
</footer>



  </body>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/assets/js/mansory.js" type="text/javascript"></script>


  


<!-- Medium Zoom JS -->
<script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
<script src="/assets/js/zoom.js"></script>


<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>


</html>
