<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>

  Adithi Rao


  | An Analysis of Butterfly Phylogenetics in Mollem National Park, India

</title>
<meta name="description" content="This is a placeholder description.
">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

<!-- Styles -->

<link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22></text></svg>">

<link rel="stylesheet" href="/assets/css/main.css">
<link rel="canonical" href="/blog/2021/Mollem/">

<!-- JQuery -->
<!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>


<!-- Theming-->






    
<!-- MathJax -->
<script type="text/javascript">
  window.MathJax = {
    tex: {
      tags: 'ams'
    }
  };
</script>
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.1.2/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
    <div class="container">
      
      <a class="navbar-brand title font-weight-lighter" href="/">
       Adithi Rao
      </a>
      
      <!-- Navbar Toggle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/">
              about
              
            </a>
          </li>
          
          <!-- Blog -->
          <li class="nav-item active">
            <a class="nav-link" href="/blog/">
              blog
              
            </a>
          </li>
          
          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          
          
          
          
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      

<div class="post">

  <header class="post-header">
    <h1 class="post-title">An Analysis of Butterfly Phylogenetics in Mollem National Park, India</h1>
    <p class="post-meta">June 2, 2021</p>
  </header>

  <div align="justify">
    <article class="post-content">
      <h3 id="introduction">Introduction</h3>

<p>In light of the recent clearance approval for Mollem National Park and Bhagwan Mahaveer Wildlife Sanctuary in Goa for infrastructural projects, the issue of biodiversity loss has become much more real. The Bhagwan Mahaveer Sanctuary is Goa’s largest protected area that falls in the Western Ghats. Located near Mollem, it is about 60km south-east of the Panaji district in Goa. It is one of the 36 biodiversity hotspots and was also recently declared a natural World Heritage Site. In addition to this, it is an important tiger corridor between Goa and the Kali Tiger Reserve, in Karnataka.</p>

<div class="container">
<div class="row">
<div class="col">
<figure class="text-center">
  <img class="center-block" src="/assets/img/Mollem_Map.webp" width="300" />
  <figcaption class="figure-caption"> <a href="https://timesofindia.indiatimes.com/india/goas-throbbing-nightlife-glowing-fungi-fireflies-and-rare-reptiles/articleshow/70153716.cms" target="blank">Source: Times of India</a> 
 </figcaption>
</figure>
</div>
</div>
</div>

<p>Infrastructural projects such as track laying and instalment of high tension wires, especially on rugged terrain, will cause a lot of damage and disturbance in this biologically important area. The approval of projects like this, requires site inspection and study, especially in the wake of climate change, which was allegedly not done <a href="https://www.sundayguardianlive.com/news/protests-goa-infra-projects-protected-area">[1]</a>. These ghats are such an old range, full of endemic species and heritage houses that are found exclusively there.</p>

<p>For this reason, we felt it was pertinent to compute the phylogenetic diversity in these areas being destructed, and in a way, see what will be lost. While the effects of climate change are countless, one that is not being recognized enough, despite its importance, is biodiversity loss. These fast paced changes in climatic and environmental conditions are posing a large challenge to a multitude of organisms, and are making it extremely difficult for them to adapt and keep up with the changes around them. There are a large number of mammals, amphibians and reptiles, birds and butterflies in this geographic area, but this report will only deal with butterflies.</p>

<p>Part of the reason I chose this group (butterflies) to study, was because they are excellent indicator organisms for biodiversity studies <a href="https://doi.org/10.1007/bf02703397">[2]</a>. They are common in most places and also easy to observe for a study of this nature. It is comparatively easier to reliably identify butterflies, both common and rare, making the data more promising. Further, since many species of caterpillars have very strict dependence on certain host flora, their diversity and community composition are dependent on the plants found in that area. They also metamorphosize, and contribute to local diversity extensively, due to their dual functionalities.</p>

<p>From pre-existing literature <a href="https://doi.org/10.1007/bf02703397">[2]</a>, we know that the Western Ghats as a whole, has butterfly fauna comprising 330 species, across 166 genera and 5 families. The goal was to construct a phylogenetic tree of the ~140 species at Bhagwan Mahaveer Sanctuary and Mollem National Park, in hopes to increase awareness and examine the potential damage that a project like this, can do.</p>

<h3 id="methods">Methods</h3>

<p>Bhagwan Mahaveer Sanctuary and Mollem National Park spans about 240 square kilometres, in the Western Ghats. Its butterfly diversity includes an estimated 143 species, including dozens of endemic species, as well. I documented the species of butterflies present here from the India Biodiversity Portal, which is an open-access repository of information on India’s biodiversity. Through this, I was able to find adequate sequences for 102 out of the 127 species listed on the checklist <a href="https://indiabiodiversity.org/checklist/show/228">[3]</a>. The gene sequences for each of the 102 species, as well as my out-groups, were obtained from GenBank <a href="https://www.ncbi.nlm.nih.gov/genbank/">[4]</a> and downloaded in Fasta format.. The phylogenetic analyses was performed based on between 650 and 1000bp of mitochondrial DNA, cytochrome oxidase subunit I (COI) gene sequence data, for all individuals.</p>

<p>Three out-groups from two genera were chosen – <em>Apis cerana indica</em>, <em>Paragomphus genei</em>, and <em>Paragomphus fritillarius</em> – to note their relative positions, from the other 102 species. Numerous butterfly phylogenetic studies <a href="https://doi.org/10.1007/bf02703397">[2]</a> in the past have used similar out- groups, and I felt that they were quite distant from the in-group, while still being adequately related – this took some trial and error. The out-group also needed to be rooted manually. It is important to note that since I had three out-groups and the software only allows you to root one, I re-made a tree with only <em>Apis indica</em> as the out-group, for my phylogenetic diversity calculation (excluded the two <em>Paragomphus</em> species).</p>

<p>To quantify the butterfly diversity in Mollem National Park and Bhagwan Mahaveer Wildlife sanctuary, I used the software MEGAX (Molecular Evolutionary Genetics Analysis), to build a phylogenetic tree, and R packages ape, caper, picante and phytools, to calculate phylogenetic diversity. MEGAX offers a variety of different settings, so that one can choose what is most fitting for their respective project. For alignment, I decided to go with the DNA Muscle alignment option. This made sense to do, since it can generate multiple alignments of amino acids and nucleotide sequences. It has a very high speed, in comparison to MAFFT, CLUSTALW, and other such options, and has the highest rank in accuracy, as well.</p>

<p>In terms of phylogenetic tree building, the choice was Maximum Likelihood Estimation. It helps provide the probabilities of the sequences, given their evolution on a particular tree. By this logic, the more probable the sequences, the more the tree is preferred. It aims to find the tree that gives you the highest probability of the observed characteristics; it is a very robust and commonly used method. This choice also allows us to infer the branch length (the time axis), which we would be unable to do with other methods like Maximum Parsimony, which would just provide the topology without any temporal information.</p>

<p>The model I used, was the General Time Reversible Model; the genome itself is not a monolith, and I felt that this method was best suited, with my limited understanding of it, and classroom discussions. Next, for the rates among sites, because not all positions of the genome evolve at the same rate, Gamma Distributed with Invariant Sites (G + I) was selected. The logic behind choosing this is that not all sites would mutate, but the ones that do, correspond to a gamma distribution. The heuristic method chosen was the default setting – nearest neighbour interchange (NNI).</p>

<p>After making these choices, I bootstrapped it, with a value of 500. This is a very important step, especially when constructing trees with not many species and not enough sequences or base pairs, for it to be computationally robust. This process gives you a value, which indicates what percentage of times, the same branch was observed in the same location, every time the tree was reconstructed in each subsequent iteration (or the same nodal relationship was recovered). In a way, it assesses confidence associated with the observed tree, at each node (or speciation event), and depicts one consensus tree.</p>

<h3 id="inferences-and-discussion">Inferences and Discussion</h3>

<div class="float-left">
<img class="img-fluid rounded" src="/assets/img/Mollem_F1.png" width="500" />
</div>

<p>Phylogenetic trees like this one (Figure 1), help one gauge the evolutionary relationship among species or other entities, that are assumed to have a common ancestor. In these trees, each node, which is a taxonomic unit, represents the most recent common ancestor of its descendants. A relatively high amount of sequences analysed (650 – 1200 bp), the use of a maximum likelihood method, and even introduction of the gamma parameter in the analysis, resulted in the phylogenetic tree shown in Figure 1. Because the method used here is Maximum Likelihood, the simplifying assumption made is that once diverged, the species evolve independently.</p>

<p>This tree was the outcome of the 500 replicate bootstrap performed, and it had the highest log likelihood value of -25696.79. Of course, this value cannot be used in isolation as an index of fit, because it is a function of sample size. However, it can be used in comparison of different coefficients and factors.</p>

<p>In this tree, the node preceding <em>Zizina otis</em> and <em>Zizeeria lysimon</em> (Figure 1, Box A) has diverged, hence they share a more recent common ancestor with each other, than either shares with any other tip in the tree. Therefore, they are also more closely related to each other than to any of the other tips. However, every other tip is equally related to both <em>Zizina otis</em> and <em>Zizeeria lysimon</em>.</p>

<p>In terms of the oldest lineage, it is clear that the clade consisting of <em>Tagiades gana</em>, <em>Cethosia nietneri</em>, <em>Graphium Agamemnon</em>, <em>Papilio polytes</em>, <em>Graphium sarpedoon</em>, <em>Chilades pandava</em> and <em>Eurema blanda</em> (Figure 1, Box E), is the oldest. This explains why is it the very last clade on this tree. Further proof of this is seen when the branch lengths are noted (as opposed to just topology), where this length corresponds to time (Figure 2). Here too, we see that this clade has the longest branch length, and hence is the oldest lineage. By the same logic as the above conclusion, the divergence event separated the lineage that gave rise to this oldest clade, from the lineage that gave rise to all the other tips in this tree (that is, <em>Vindula erota</em> and above).</p>

<p>The phylogenetic diversity (PD) value was also calculated, using the pd.calc() function in R. For the purpose of this, another tree was used, with only <em>Apis cerana indica</em> as the out-group. The two <em>Paragomphus</em> species were removed, as only one out-group could be rooted in this software. This resulted in a total of 102 species, excluding the out-groups. The value obtained was 118.97, which is arbitrary by itself. Since PD is not a fundamental biological unit, it cannot be analysed in isolation. This value is only of use when compared with others. For example, a similar phylogenetic tree for the 33 mammals of Bhagwan Mahaveer Sanctuary, yielded a PD value of 10.19. So from this we can say that the phylogenetic diversity is higher for butterflies, than it is for mammals. This also seems to be correlated to the number of species considered, which makes sense, since more species, especially if spanning multiple genera, would mean more diversity. It also seems to vary with bootstrap number – two other individuals with similar butterfly trees, got PD values of 125.1485 and 182.0359. The value of this tree, too, changed from 118.97 to 231.5219, when bootstrap number was reduced, which is strange. It could point to a lack computational robustness, or just a lack of enough data.</p>

<div class="float-right">
<img class="img-fluid rounded" src="/assets/img/Mollem_F2.png" width="500" />
</div>

<p>On looking at the generated tree in Figure 1, we can see a few discrepancies. For one, it seems that a lot of nodes have values much lower than 100 and some don’t seem to have an associated value at all. This could be due to multiple reasons. One, insufficient data – between 650 and 1200 bp of nucleotides were used, which might not be sufficient for a group that is significantly genetically similar. We can also see that out-group 1 (Figure 1) does not have an associated value; this is because it was manually rooted by force. Another reason for some clades having no bootstrap values could be potentially because more than one clade was equally distant from other groups, so the software was not able to assign numbers effectively. It is also possible that a low bootstrap value means that those species do not belong together in the same clade, for example, Figure 1, Box B.</p>

<p>It is also strange that the monophyletic <em>Paragomphus</em> out-groups (Figure 1, out-groups 2a, 2b) are depicted as a part of the in-group, despite being quite distantly related to butterflies. This could again be due to the fact that there was not enough data used to build the tree. It could again also be a lack of computational robustness.</p>

<p>There are multiple clades wherein the bootstrap values are really low for example, Box B in Figure 1. However, in contrast, Box C and D show relatively high bootstrap values, which shows that it is likely that they belong in that position, in the clade that they are seen in, since a high percentage of total iterations pointed to the same result.</p>

<p>In terms of history, this range is older than the Himalayan mountain chain and has a large influence on the biophysical and ecological processes over the entire peninsula of India, and even abroad. The vegetation and forest ecosystems in the Western Ghats play a large role in the Indian monsoon patterns, and mediate and channel the tropical climate found here. In addition to this, by virtue of sheer size alone, they act as a physical barrier and intercept the monsoon winds that travel from south to west, every year in summer. Traditional climatology is often based on boundaries and barriers, which modulates factors like wind, water and air. Since the Western Ghats are on the elevated corner of the Deccan Plateau, it is that much more central to the climate patterns we observe. These Ghats are not actual mountains in the true sense, but rather a faulted edge of a plateau that has been upraised. They are denuded, not deformed, with ravines and canyons along the scrap that faces the Arabian sea, with valleys on the east.</p>

<p>In the recent years, the focus on nature conservation efforts has increased rapidly. Biological diversity is recognized as a crucial parameter to monitor local, as well as global environmental and climatic changes. Along with this, since the Western Ghats are so diverse in range, topography and biogeography, they have been subjected to a variety of conservation efforts. These Ghats are home to multiple major Indian passes, such as the Goa Gap, which sits between Karnataka and Maharashtra; it is a climate-mediated ecological divide <a href="https://doi.org/10.1111/jav.01278">[5]</a>. These gaps are dispersal barriers, which are key in often preventing organisms from relocating, serving as a mode through which speciation occurs, and diversity increases <a href="https://www.nature.com/scitable/knowledge/library/causes-and-consequences-of-dispersal-in-plants-15927714/">[6]</a>. It is also the perfect microhabitat for a large range or organisms that are found here.</p>

<p>In terms of butterfly diversity, the Western Ghats harbour 330 species, including 37 that are endemic, and another 23 that are shared only with Sri Lanka. These butterfly species span across 166 genera and 5 families! The butterfly species richness was relatively high for the natural forests which had a larger amount of tree species. The numbers were highest for deciduous forests, and rather low for evergreen and semi-evergreen forests, followed by scrub/savanna. This could be because these forests lack their own distinctive species, and attract generalists, instead <a href="https://doi.org/10.1007/bf02703397">[2]</a>. Though butterfly endemicity was found to be approximately 10% in this area <a href="http://wgbis.ces.iisc.ernet.in/biodiversity/sahyadri_enews/newsletter/issue53/bibliography/115-a-brief-overview-of-the-western-ghats.PDF">[7]</a>, it is much less than other groups of organisms that are found here.</p>

<p>The evergreen forests had high alpha and beta diversity values, which is expected in view of the potential shared, wide-range species. Since it is peak monsoon-land, the vegetation and plant diversity here is mostly a product of seasonal rainfall, as opposed to the atmospheric temperature <a href="https://doi.org/10.1007/978-94-010-2331-3_7">[8]</a> – this plant diversity in turn affects the butterfly species found.</p>

<p>From this, and the hundreds of studies done in the past, it is clear that the Western Ghats in India, being a perfect example of non-equatorial tropical evergreen forest, is home to at least 325 globally threatened organisms <a href="https://whc.unesco.org/en/list/1342/">[9]</a>, spanning flora and fauna, both. In an era like this one, more work should go into taking care of this diversity, as opposed to getting rid of it. Biodiversity loss is just as big an issue as climate change, and they’re both intimately related. Both have to be taken care of, to avoid such precious landmarks from crossing their tipping point and being destroyed. The Western Ghats have played such a crucial role on the Indian Peninsula, and a lot of what the subcontinent is like now, can be attributed to this range.</p>


    </article>
  </div>

  

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    &copy; Copyright 2021 Adithi  Rao.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>.

    
    
  </div>
</footer>



  </body>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/assets/js/mansory.js" type="text/javascript"></script>


  


<!-- Medium Zoom JS -->
<script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
<script src="/assets/js/zoom.js"></script>


<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>


</html>
